src/mudlet.cpp

```diff
@@ -1413,6 +1422,14 @@ void mudlet::slot_tab_changed(int tabID)
 
     dactionInputLine->setChecked(mpCurrentActiveHost->getCompactInputLine());
 
+    // If game has custom invite then make secondary Discord option visible.
+    bool isVis = mpActionMudletDiscord->isVisible();
+    if ( mpCurrentActiveHost->getDiscordInviteURL().isEmpty() && isVis ) {
```

Comment:
i suggest doing the computationally simplest test first! Also our coding style does not put spaces between the parenthesis of the `if` and what they contain - so this line would become:
```suggestion
    if (isVis && mpCurrentActiveHost->getDiscordInviteURL().isEmpty()) {
```